core/dvector.h

```diff
@@ -77,7 +77,7 @@ struct MemoryPool {
 	static void cleanup();
 };
 
-/**
+/*
 	@author Juan Linietsky <reduzio@gmail.com>
 */
```

Comment:
Not exactly related to this PR, but since you're editing the file...

I'm unsure why this `@author` comment is here, especially with the copyright notice already at the top of the file. It seems kinda redundant and unneeded. At the very least, I think it should be a `// comment` since it's only the one line.

Reply:
This author comment appears in multiple files that I've seen.

I do think it is 'kinda redundant and unneeded' and would clean up the code. Although it is down to @reduz's discretion as to whether he would be fine with:
(1) Removal of the author comment.
or
(2) Make it a single line comment (i.e. reduce its importance).
or
(3) Keep it exactly how it is.

I'd be happy to change it in all files I refactor, if I knew his preference.